packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt

```diff
@@ -0,0 +1,70 @@
+package expo.modules.video
+
+import android.content.Context
+import androidx.media3.common.MediaItem
+import androidx.media3.common.Player
+import androidx.media3.common.Timeline
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.exoplayer.ExoPlayer
+import expo.modules.kotlin.sharedobjects.SharedObject
+import java.io.Closeable
+
+// https://developer.android.com/guide/topics/media/media3/getting-started/migration-guide#improvements_in_media3
+@UnstableApi
+class VideoPlayer(context: Context, private val mediaItem: MediaItem) : Closeable, SharedObject() {
```

Comment:
```suggestion
class VideoPlayer(context: Context, private val mediaItem: MediaItem) : AutoCloseable, SharedObject() {
```
It's better because you can use it with the `.use` function. It's probably not needed for now, but it may be handy if we add tests. But I prefer using `AutoCloseable` instead of `Closeable`